nextflow.config

```
@@ -198,10 +198,10 @@ try {
 // Load nf-core/taxprofiler custom profiles from different institutions.
 // Warning: Uncomment only if a pipeline-specific instititutional config already exists on nf-core/configs!
```

```suggestion
// Warning: Uncomment only if a pipeline-specific institutional config already exists on nf-core/configs!
```

This is what the template was trying to address

---

Do you mean that I shall not keep this one?

---

Just commit my suggestion :)
(template has all those lines commented out, the conflict because we uncomment the if/else statement - but we need to also fix the typo)